### ISSUE TITLE
trying to run css task

### ISSUE THREAD

**Issue by dv336699** [booms8/gulp-task-launcher#2]
``` javascript

gulp.task('compile:less', function () {
    return gulp.src('src/less/main.less')
        .pipe(less({
            paths: [path.join(__dirname, 'less', 'includes')]
        }))
        .pipe(rename(pkg.css))
        .pipe(gulp.dest('./bin/assets/'));
});

gulp.task('compile:css', ['compile:less'], function () {
    var dest = config.vendor_files.css;
    dest = dest.concat(['./bin/assets/' + pkg.css]);

    return gulp.src(dest)
        .pipe(concat(pkg.css))
        .pipe(minifyCSS())
        .pipe(gulp.dest('./bin/assets/'));
});
```

**Atom Version**: 1.5.3
**System**: Mac OS X 10.11.3
**Thrown From**: [gulp-task-launcher](https://github.com/booms8/gulp-task-launcher) package, v1.4.6
### Stack Trace

Uncaught Error: Syntax error, unrecognized expression: unsupported pseudo: css

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:1458

Error: Syntax error, unrecognized expression: unsupported pseudo: css
    at Function.Sizzle.error (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:1458:8)
    at Sizzle.selectors.filter.PSEUDO (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:1765:13)
    at matcherFromTokens (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:2308:44)
    at Sizzle.compile (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:2450:13)
    at Sizzle.select (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:2536:16)
    at Function.Sizzle [as find] (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:855:9)
    at gulpTaskLauncherView.jQuery.fn.extend.find (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:2695:11)
    at gulpTaskLauncherView.module.exports.gulpTaskLauncherView.runGulp (/Users/diego/.atom/packages/gulp-task-launcher/lib/gulp-task-launcher-view.coffee:196:10)
    at gulpTaskLauncherView.module.exports.gulpTaskLauncherView.run (/Users/diego/.atom/packages/gulp-task-launcher/lib/gulp-task-launcher-view.coffee:168:10)
    at space-pen-div.<anonymous> (/Users/diego/.atom/packages/gulp-task-launcher/lib/gulp-task-launcher-view.coffee:40:33)
    at space-pen-div.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
    at space-pen-div.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
```
### Commands

```
     -0:59.3.0 core:close (atom-text-editor.editor.is-focused)
     -0:56.5.0 find-and-replace:show (atom-text-editor.editor.is-focused)
  3x -0:55.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:53.9.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:53.9.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:50.3.0 find-and-replace:show (atom-text-editor.editor.is-focused)
  2x -0:48.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:47.5.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:47.5.0 core:cancel (atom-text-editor.editor.mini.is-focused)
  4x -0:45.6.0 find-and-replace:find-next (atom-text-editor.editor.is-focused)
     -0:32 find-and-replace:show (atom-text-editor.editor.is-focused)
  5x -0:29.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:26.8.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:26.8.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:13 tree-view:remove (span.name.icon.icon-file-text)
     -0:09.6.0 core:close (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {},
  "gulp-task-launcher": {
    "build": true,
    "build:api": true,
    "build:api:vendor": true,
    "build:assets": true,
    "build:clean": true,
    "build:css": true,
    "build:extra": true,
    "build:fonts": true,
    "build:html2js": true,
    "build:index": true,
    "build:js": true,
    "build:js_vendor": true,
    "build:jshint": true,
    "build:karma-config": true,
    "build:less": true,
    "build:replace:versao": true,
    "build:test": true,
    "bump": true,
    "bump:major": true,
    "bump:minor": true,
    "compile": true,
    "compile:api": true,
    "compile:assets": true,
    "compile:clean": true,
    "compile:concat": true,
    "compile:css": true,
    "compile:db": true,
    "compile:extra": true,
    "compile:fonts": true,
    "compile:html2js": true,
    "compile:index": true,
    "compile:js": true,
    "compile:karma-config": true,
    "compile:less": true,
    "compile:post-clean": true,
    "compile:replace:versao": true,
    "compile:test": true,
    "compile:uglify": true,
    "default": true,
    "tasks": [
      "build",
      "build:api",
      "build:api:vendor",
      "build:assets",
      "build:clean",
      "build:css",
      "build:extra",
      "build:fonts",
      "build:html2js",
      "build:index",
      "build:js",
      "build:js_vendor",
      "build:jshint",
      "build:karma-config",
      "build:less",
      "build:replace:versao",
      "build:test",
      "bump",
      "bump:major",
      "bump:minor",
      "compile",
      "compile:api",
      "compile:assets",
      "compile:clean",
      "compile:concat",
      "compile:css",
      "compile:db",
      "compile:extra",
      "compile:fonts",
      "compile:html2js",
      "compile:index",
      "compile:js",
      "compile:karma-config",
      "compile:less",
      "compile:post-clean",
      "compile:replace:versao",
      "compile:test",
      "compile:uglify",
      "default",
      "watch"
    ],
    "watch": true
  }
}
```
### Installed Packages

``` coffee
# User
Remote-FTP, v0.7.10
aligner-php, v1.0.0
angularjs, v0.3.4
atom-autocomplete-php, v0.18.7
atom-beautify, v0.28.19
autocomplete-ionic-framework, v0.2.2
autocomplete-ionic2-framework, v0.3.2
autocomplete-php, v0.3.7
bottom-dock, v0.3.7
build, v0.55.0
build-gulp, v0.7.0
drunken-php, v0.2.1
editorconfig, v1.2.2
emmet, v2.4.1
file-icons, v1.6.11
file-type-icons, v0.7.3
gulp-manager, v0.2.18
gulp-task-launcher, v1.4.6
ionide-webview, v1.0.3
linter, v1.10.0
linter-php, v1.1.0
php-analyser, v0.2.0
php-composer-completion, v0.3.4
php-cs-fixer, v2.5.1
php-integrator-annotations, v0.3.1
php-integrator-autocomplete-plus, v0.5.2
php-integrator-linter, v0.1.0
php-integrator-navigation, v0.5.0
php-integrator-tooltips, v0.4.1

# Dev
No dev packages
```

**Comment by booms8** [comment]
Accidentally removed the escape for colons in task names. Should be fixed now (1.4.9).

**Comment by dv336699** [comment]
@booms8 awesome, thanks for that
